controllers/bpfman-operator/common_cluster.go

```diff
@@ -74,7 +74,7 @@ func statusChangedPredicateCluster() predicate.Funcs {
 		UpdateFunc: func(e event.UpdateEvent) bool {
 			oldObject := e.ObjectOld.(*bpfmaniov1alpha1.ClusterBpfApplicationState)
 			newObject := e.ObjectNew.(*bpfmaniov1alpha1.ClusterBpfApplicationState)
-			return !reflect.DeepEqual(oldObject.GetStatus(), newObject.Status)
+			return !reflect.DeepEqual(oldObject.GetStatus(), newObject.Status.Conditions)
```

Comment:
Need a similar change in controllers/bpfman-operator/common_namespace.go.

Reply:
That may have been why the integration test cleanup was failing.